ja-jp/Content/Topics/ReleaseNotes/mobile-release-status.htm

```diff
@@ -304,669 +303,10 @@
-                                                                    </div>
-                                                                </div>
-                                                                <div class="MCDropDown MCDropDown_Closed dropDown" style="display: none;" data-mc-target-name="rel-hist-ios"><span class="MCDropDownHead dropDownHead"><a href="#" class="MCDropDownHotSpot dropDownHotspot MCDropDownHotSpot_ MCHotSpotImage" aria-expanded="false"><img class="MCDropDown_Image_Icon" src="../../../Skins/Default/Stylesheets/Images/transparent.gif" height="11" width="16" alt="閉じた" data-mc-alt2="開く" />Okta Mobile 5.6.0 for iOS</a></span>
-                                                                    <div class="MCDropDownBody dropDownBody">
-                                                                        <p><b>Okta Mobile 5.6.0 for iOS</b>は<b>2017年6月13日</b>にリリースされました。<b>Apple App Store</b>で入手できます。</p>
-                                                                        <p>このリリースには、以下を含む一般的なパフォーマンスの改善とバグ修正が含まれています。</p>
-                                                                        <h3>新機能</h3>
-                                                                        <ul>
-                                                                            <li>
-                                                                                <p>プライベート・アプリの更新がありますか？簡単にプライベート・アプリを更新し、<b>Okta Mobileアプリ・ストア</b>を通じてユーザーに配布できるようにしました。 </p>
-                                                                            </li>
-                                                                            <li><b>[インストールされているアプリケーション]</b>を独自のページに含め、アンインストールされたアプリ、更新が必要なアプリ、またはOMMで保護されていないアプリの表示をそのままにすることで、ユーザー・エクスペリエンスを改善しました。                        </li>
-                                                                        </ul>
-                                                                        <h3>バグ修正</h3>
-                                                                        <ul>
-                                                                            <li>OKTA-118559：iOSのパスワード有効期限切れエラー・メッセージで、ブラウザーからパスワードを変更する必要があることを指示できませんでした。                        </li>
-                                                                        </ul>
-                                                                    </div>
-                                                                </div>
-                                                            </div>
-                                                        </div>
-                                                    </div>
+                                                    <h1 class="relnotes">未翻訳のドキュメント</h1>
+                                                    <p>このページは、現在まだ翻訳されていません。最新のリリース・ノートについては、
+			<a href="https://help.okta.com/en-us/Content/Topics/ReleaseNotes/okta-relnotes.htm">リリース・ノート（英語版）</a>

```

Comment:
similar question here to what I made in the info-dev PR. Any reason we don't have this link to https://help.okta.com/en-us/Content/Topics/ReleaseNotes/mobile-release-status.htm instead?

Reply:
here and elsewhere, revised these links to point to their EN equivalent